client/src/components/details/DetailsContainer.tsx

```diff
@@ -54,7 +54,7 @@ class DetailsContainer extends Component<Props, State> {
     this.setState({
       isLoading: true,
     });
-    fetch(`http://172.31.44.202:5000/api/devices/ProszeMiPoRazKolejnyTegoNieUsuwac/tasks/${_id}`)
+    fetch(`http://192.168.1.105:5000/api/devices/ProszeMiPoRazKolejnyTegoNieUsuwac/tasks/${_id}`)
```

Comment:
XD ktoś to na pewno zmieni prędzej czy później 😆

Reply:
nooo ja, ale inne rzeczy mają moim zdaniem większy priorytet XD

Reply:
a to z tym aws też nie takie proste :(